app/services/statistics_calculator.rb

```diff
@@ -1,37 +1,86 @@
 class StatisticsCalculator
 
+    module Constants
+        NUMBER_OF_QUESTIONS = 10
+        FULL_PERCENTAGE = 100
+    end
+    
+    module Score
+        A = 90
+        B = 75
+        C = 60
+    end
+    
     class << self 
 
         def success_per_question(results)
             statistics = Array.new;
-            questions = Array.new(10,0);
+            questions = Array.new(Constants::NUMBER_OF_QUESTIONS, 0);
             
             results.each {|result| count_correct_answers_in_result(result.answers, questions)}
             claculate_correct_answers_percentage(questions, results.length(), statistics)
             statistics
+        end
 
+        def user_score(results)
+            statistics = Array.new;
+            results.each {|result| count_correct_answers_for_user(result.answers, result.user, statistics)}
+            statistics
```

Comment:
```suggestion
            results.map { |result| count_correct_answers_for_user(result.answers, result.user) }
```
You don't need to pass statistics inside, just map the results which will automatically return an array
This way you don't send a param which you change inside the function